src/test/java/com/andreamazzarella/contact_manager/helpers/InMemoryRepositoryHelper.java

```diff
@@ -0,0 +1,19 @@
+package com.andreamazzarella.contact_manager.helpers;
+
+import com.andreamazzarella.contact_manager.Contact;
+import com.andreamazzarella.contact_manager.InMemoryRepository;
+
+import java.util.HashMap;
+
+public class InMemoryRepositoryHelper {
+    public static Contact createContactInRepository(InMemoryRepository repo, String[] contactInformation) {
+        HashMap<String, String> contactMap = new HashMap<>();
+        for (int key = 0; key < contactInformation.length; key += 2) {
+            contactMap.put(contactInformation[key], contactInformation[key + 1]);
```

Comment:
With types, you can do `contactInformation.getFirstname(), contactInformation.getLastName()` which is a little easier to understand.